pkg/api/api_test.go

```diff
@@ -88,6 +89,23 @@ func newTestServer(t *testing.T, o testServerOptions) (*http.Client, *websocket.
 	return httpClient, conn, ts.Listener.Addr().String()
 }
 
+func request(t *testing.T, client *http.Client, method, resource string, body io.Reader, responseCode int) *http.Response {
```

Comment:
`checkResponseCode` or  something better

Reply:
not just, since it also returns a `Response`

Reply:
`testedRequest`?